scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1947,65 +1961,80 @@ private void sendMessage(final long nanos, final RawData message, final Connecti
 				initializeHandshaker(handshaker);
 				handshaker.startHandshake();
 			}
+			message.onConnecting();
```

Comment:
I have no idea but is this `onConnecting` should be add in else block below ?

Reply:
This message is sent while the handshake. For me this is right.